Dapper/ExplicitConstructorAttribute.cs

```diff
@@ -5,7 +5,7 @@ namespace Dapper
     /// <summary>
     /// Tell Dapper to use an explicit constructor, passing nulls or 0s for all parameters
     /// </summary>
-    [AttributeUsage(AttributeTargets.Constructor, AllowMultiple = false)]
+    [AttributeUsage(AttributeTargets.Constructor | AttributeTargets.Method, AllowMultiple = false)]

```

Comment:
Can we add a comment noting that usage on methods is limited to usage with DapperAOT?

Reply:
added a noting comment